imageroot/actions/get-configuration/20readconfig

```diff
@@ -8,6 +8,30 @@
 import json
 import os
 import sys
+import agent
+import subprocess
+
+rdb = agent.redis_connect() # full read-only access on every key
+modules=[]
+accepted_timezone_list = []
+list_of_timezone_widget = []
+
+# we query about all mail server to use it inside the user interface
+for key in agent.list_service_providers(rdb,'imap','tcp'):
+    modules.append({
+        "name": key['module_id'],
+        "label": key['mail_hostname'],
+        "value": key['module_id'],
+    })
+# we query about all known timezone and we build a widget
+accepted_timezone_list = subprocess.check_output(['podman', 'run', '--rm', os.environ["WEBTOP_WEBAPP_IMAGE"], 'java', '-classpath', '/usr/share/webtop/', 'ListTimeZones'], text=True).splitlines()
+
+for value in accepted_timezone_list:
+    list_of_timezone_widget.append({
+        "name": value,
+        "label": value,
+        "value": value,
+    })
```

Comment:
Instead of changing the `get-configuration` is better to add a new action, see the comment below.